hack/install-pipeline-service.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash -e
+
+ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/..
+
+source ${ROOT}/hack/flags.sh "The install-pipeline-service.sh installs Pipeline Service for development and testing on non-production clusters / kcp instances."
+MODE=preview parse_flags $@
+
+
+PIPELINE_SERVICE_DIR=$(mktemp -d)
+git clone -b record-server-ca-during-registration --depth 1 https://github.com/lcarva/pipeline-service/ $PIPELINE_SERVICE_DIR
```

Comment:
FYI, this is in main now: https://github.com/openshift-pipelines/pipeline-service/pull/245